### PR TITLE
Quiet Cryo Pods

### LINKED ISSUE #12478: Ghost Cafe annouces cyros
probably shouldn't alert the station whenever someone leaves the ghost cafe

### PR DESCRIPTION
**Pull request by Quill-Weave**
image

## About The Pull Request

Fixes: #12478

Adds quiet cryo pods that do not announce when a crewmember goes into cryogenics to the station.
Currently only added to the Ghost Cafe, as it's the only documented place where this problem comes up to my knowledge.
The code is simple and in the future, other cryo pods can be fixed if they are an issue.
The Cryogenics Oversight Console still stores the names and identities of people who cryo in quiet pods for the sake of administration.

EDIT: Added to multiple locations due to request.

## How This Contributes To The Skyrat Roleplay Experience

Prevents the station crew from hearing about snuggle buddies who don't exist (contrary to the snuggle buddies who don't exist already) going to cryo. eg; Ghost cafe people. 

In addition, opens the door to future coding for RP-friendly ways to cryo that might not notify the station.

## Changelog

:cl:
fix: replaces all cryo pods in the ghost cafe, syndicate base, port tarkon and DS-2 with quiet cryo pods
code: adds quiet variable/version to cryo pods
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/cryosleep/code/cryopod.dm [hunk 1]
```diff
@@ -134,7 +134,11 @@ GLOBAL_LIST_EMPTY(cryopod_computers)
 	///Weakref to our controller
 	var/datum/weakref/control_computer_weakref
 	COOLDOWN_DECLARE(last_no_computer_message)
+	var/quiet = FALSE
 
+
+/obj/machinery/cryopod/quiet
+	quiet = TRUE
```
Comment (Zonespace27) [inline modular_skyrat/modules/cryosleep/code/cryopod.dm:142]:
```suggestion
	quiet = TRUE
	
```

File: modular_skyrat/modules/cryosleep/code/cryopod.dm [hunk 2]
```diff
@@ -134,7 +134,11 @@ GLOBAL_LIST_EMPTY(cryopod_computers)
 	///Weakref to our controller
 	var/datum/weakref/control_computer_weakref
 	COOLDOWN_DECLARE(last_no_computer_message)
+	var/quiet = FALSE
```
Comment (Zonespace27) [inline modular_skyrat/modules/cryosleep/code/cryopod.dm:138]: code doc comment this (`/// what you want the comment to be` above the `var/quiet` code)

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/cryosleep/code/cryopod.dm (hunk 2 above)
Comment (Quill-Weave) [reply modular_skyrat/modules/cryosleep/code/cryopod.dm:138]: Done! Code modified as requested!

**Review by Zonespace27 — CHANGES_REQUESTED** [review]
Also please change the lavaland syndicate base and port: tarkon cryo pods to this

**Comment by YakumoChen** [comment]
changelog should be reduced to one CL. Given this fixes a bug, it should be a fix: prefix, eg 

fix: ghost role cyropods no longer announce their function to the station.

**Comment by Suspect-One** [comment]
You should add them to DS-2 too.

**Comment by YakumoChen** [comment]
There's no edit: type CL. Prefix it with fix:

Code: is for code improvements. Generally this does not see use as code improvements are not often player-facing (which is what the changelog is meant for)

**Pull request merged**